appscale/tools/templates/AppScalefile

```diff
@@ -23,19 +23,27 @@ ips_layout:
       - zookeeper
     nodes: [1 | vm1_private_IP]
 
-# A multi node deployment.
+# A multi node deployment example. Within nodes one can use the numbers of
```

Comment:
numbers -> number 
Maybe simplify it to "Within nodes, one can specify either the number of nodes desired (cloud deployment) or a list of IPs (cluster deployment)."